requirements.txt

```diff
@@ -1,6 +1,7 @@
 # Versions are pinned to prevent pypi releases arbitrarily breaking
 # tests with new APIs/semantics. We want to update versions deliberately.
 ansible==2.2.2.0
+boto==2.45.0
```

Comment:
Perhaps not that bad. It means that whoever uses `pip install -r requirements.txt` will get `boto` even if they don't use it... but then, we might already have or will have other cases like this -- dependencies that are related only to a specific usage of the project.

:+1: to adding it to `requirements.txt` and keeping the image build simple.